source/user-manual/reference/ossec-conf/wodle-osquery.rst

```diff
@@ -125,11 +125,19 @@ pack
 
 Add a query pack to the configuration. This option can be defined multiple times.
 
-+--------------------+----------------------+
-| **Default value**  | Empty                |
-+--------------------+----------------------+
-| **Allowed values** | Any available pack   |
-+--------------------+----------------------+
++--------------------+---------------------------------+
+| **Default value**  | Empty                           |
++--------------------+---------------------------------+
+| **Allowed values** | Path to pack configuration file |
++--------------------+---------------------------------+
+
+Attributes:
+
++----------+----------------------+
+| **name** | Name for this pack   |
++----------+----------------+-----+
```

Comment:
Please remove the `---------` under `**name**`, this is not generating a proper table when compiling. [More info](http://docutils.sourceforge.net/docs/user/rst/quickref.html#tables).